web3_proxy/src/lib.rs

```diff
@@ -30,4 +31,5 @@ pub struct PostLoginQuery {
 pub struct PostLogin {
     sig: String,
     msg: String,
+    pub referral_code: Option<String>,
```

Comment:
@WyseNynja was not sure how to retrieve the referral_code from the other struct, do you think leaving it here is fine? happy to also move it